homeassistant/components/light/lifx.py

```diff
@@ -265,6 +269,9 @@ def async_turn_on(self, **kwargs):
     @asyncio.coroutine
     def async_turn_off(self, **kwargs):
         """Turn the device off."""
+        if not self.available:
+            return
+
```

Comment:
No update is needed, otherwise the device have no access to come back to available state. On update you need check if the device back to controll or stay available until next update call. But update is only needed on a polling device. If you have a callback and no polling device, you can update the property on callback function.